ros2pkg/ros2pkg/resource/ament_python/setup.py.em

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup
+
+package_name = '@project_name'
+
+setup(
```

Comment:
Should there be dependencies in here?

Definitely should copy in description/other information from package.xml.